tests/unit/objectives/test_single_objectives.py

```diff
@@ -81,7 +92,7 @@ def test_objective_maps_minimizers_to_minimum(
 ) -> None:
     objective_values_at_minimizers = objective(minimizers)
     tf.debugging.assert_shapes([(objective_values_at_minimizers, [len(minimizers), 1])])
-    npt.assert_allclose(objective_values_at_minimizers, tf.squeeze(minimum), atol=1e-4)
+    npt.assert_allclose(objective_values_at_minimizers, tf.squeeze(minimum), atol=2e-4)
```

Comment:
How many samples do you need to get within the existing tolerance?

Reply:
There aren't any samples here?

Reply:
My bad. So how come we are outside tolerance then???